DirectProgramming/DPC++FPGA/Tutorials/Tools/experimental/platform-designer/add-quartus/AddCSRDemo.qpf

```diff
@@ -0,0 +1,31 @@
+# -------------------------------------------------------------------------- #
```

Comment:
I'm hopeful that we can instruct users how to open a new quartus project rather than shipping this file

Reply:
Personally, I felt that 'creating a Intel® Quartus® Prime project' is self-explanatory enough that we don't need to walk a user through it. there are a decent number of steps in the wizard and I thought it would just make this (already long) readme much longer.

Reply:
Turns out in the original HLS tutorial I did instruct how to create the project from scratch, i'll add those instructions.